Manually add the include directory of VTK
dirty fix for issue https://github.com/InsightSoftwareConsortium/ITK/issues/4375
author: Alex Domingo (Vrije Universiteit Brussel)
--- Wrapping/CMakeLists.txt.orig	2023-12-21 13:41:44.845008000 +0100
+++ Wrapping/CMakeLists.txt	2023-12-21 13:42:14.794328946 +0100
@@ -112,6 +112,7 @@
 ###############################################################################
 # Configure specific wrapper modules
 ###############################################################################
+include_directories("$ENV{EBROOTVTK}/include/vtk-9.3/")
 
 unset(WRAP_ITK_MODULES CACHE)
 
